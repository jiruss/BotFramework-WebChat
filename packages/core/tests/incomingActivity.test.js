import createFacility from './createFacility';

test('Incoming activity', async () => {
  const { actions, directLine, store } = await createFacility();

  // Clear out actions
  actions.splice(0);

  directLine.activity.next({
    text: 'Hello, World!',
    type: 'message'
  });

  await 0;

  expect(store.getState().activities).toMatchSnapshot();
  expect(actions).toMatchSnapshot();
});
