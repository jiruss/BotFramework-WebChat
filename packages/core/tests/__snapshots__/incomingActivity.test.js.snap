// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Incoming activity 1`] = `
Array [
  Object {
    "text": "Hello, World!",
    "type": "message",
  },
]
`;

exports[`Incoming activity 2`] = `
Array [
  Object {
    "payload": Object {
      "activity": Object {
        "text": "Hello, World!",
        "type": "message",
      },
    },
    "type": "DIRECT_LINE/INCOMING_ACTIVITY",
  },
]
`;
